metal/utils.py

```diff
@@ -365,3 +365,13 @@ def slice_data(data, indices):
             return outputs[0]
         else:
             return outputs
+
+
+def mt_to_cuda(data):
```

Comment:
I think we need a better name here. I didn't now what "mt" meant. I mean, I assumed multitask, but what does it mean to convert a 'multitask' to cuda?

Can we also add a docstring here saying what the expected type of "data" is?

Reply:
Also, this current function won't cover `X` that comes in as a list, right?  Will clean up now